com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -369,7 +370,7 @@ private void SpawnInternal(bool destroyWithScene, ulong ownerClientId, bool play
                 throw new NotServerException($"Only server can spawn {nameof(NetworkObject)}s");
             }
 
-            NetworkManager.SpawnManager.SpawnNetworkObjectLocally(this, NetworkManager.SpawnManager.GetNetworkObjectId(), false, playerObject, ownerClientId, destroyWithScene);
+            NetworkManager.SpawnManager.SpawnNetworkObjectLocally(this, NetworkManager.SpawnManager.GetNetworkObjectId(), IsSceneObject.HasValue && IsSceneObject.Value == true, playerObject, ownerClientId, destroyWithScene);
```

Comment:
Is the `== true` really needed? If it's boolean, the `&&` should work already, or is there some funky casting going on?

Reply:
This was probably just a mistake on my part... will remove the == true as it is not needed at all.